package.json

```diff
@@ -76,7 +76,7 @@
     "webpack-merge": "^3.0.0"
   },
   "scripts": {
-    "prepublish": "npm run clean && npm run build && npm run test",
+    "prepare": "npm run clean && npm run build && npm run test",
```

Comment:
@simark do you use the latest npm? I don't, probably we all should update to make use of `prepublishOnly` and `prepare`.